api/swagger.yaml

```diff
@@ -6302,6 +6315,75 @@ definitions:
     additionalProperties:
       type: "string"
 
+  ImageManifestSummary:
+    x-nullable: false
+    description: |
+      ImageManifestSummary represents a summary of an image manifest.
+    type: "object"
+    required: ["Id", "Descriptor", "Available", "ContentSize", "Kind"]
+    properties:
+      Id:
+        description: |
+          ID is the content-addressable ID of an image and is the same as the
+          digest of the image manifest.
+        type: "string"
+        example: "sha256:95869fbcf224d947ace8d61d0e931d49e31bb7fc67fffbbe9c3198c33aa8e93f"
+      Descriptor:
+        $ref: "#/definitions/OCIDescriptor"
+      Available:
+        description: Indicates whether all the children content is available.
+        type: "boolean"
+        example: true
+      ContentSize:
+        description: |
+          The size of the available distributable (possibly compressed) image content
+          in bytes.
+        type: "integer"
+        format: "int64"
+        example: 3987495
+      Kind:
+        description: |
+          The kind of the manifest.
+        type: "string"
+        example: "image"
+        enum:
+          - "image"
+          - "attestation"
+      ImageData:
+        description: |
+          The image data for the image manifest.
+          This field is only populated when Kind is "image".
+        type: "object"
+        required: ["Platform", "Containers", "Size", "UnpackedSize"]
+        properties:
+          Platform:
+            $ref: "#/definitions/OCIPlatform"
+          Containers:
+            description: |
+              The number of containers that are using this specific platform image.
+            type: "integer"
+            format: "int64"
+            example: 2
```

Comment:
Similar to my other question, couldn't this instead be a list of which ones? :eyes: (IDs or names, maybe both?)

Reply:
Hmm, we're already traversing all containers to determine that, so perhaps we could include that here. My only concern is that it might be _too much_ information for the "list" operation?

Reply:
I'm fine either way, because we already have that information at hand. Just slightly worried about `list` becoming an `inspect` 😅.

Reply:
> Just slightly worried about list becoming an inspect 😅.

Yeah, I can see where you're coming from. I think it may be fine to use a list for this here. Or at least, it makes the content probably a lot more useful than just a number 🤔 (but happy to discuss)